lab-anna/app/entry.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+require('./scss/reset.scss');
+require('./scss/main.scss');
+
+const angular = require('angular');
+const cowsay = require('cowsay-browser');
+
+const cowsayApp = angular.module('cowsayApp', []);
+
+cowsayApp.controller('CowsayController', ['$log', '$scope', CowsayController]);
+
+function CowsayController($log, $scope) {
+  $log.debug('CowsayController');
+
+  let cowsayCtrl = $scope.cowsayCtrl= {};
+
+  cowsayCtrl.title = 'Welcome to Cowville!';
+
+  cowsayCtrl.speak = function(input) {
+    $log.debug('cowsayCtrl.speak()');
+    return cowsay.say({ text: input || 'moooooooo' });
+  };
+
+  cowsayCtrl.logger = function(input) {
+    $log.debug('cowsayCtrl.logger()');
+    $log.log(input);
+  };
```

Comment:
Please add another functionality... maybe the 'undo' functionality we created in class to meet the assignment requirements.